node/.vtexignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
I believe this file is unnecessary, the root `.vtexignore` file should already cover this.

Reply:
I will remove this file, I put just to try to pass the [vtexio/build](https://master--vtex.myvtex.com/admin/builder-log/build-log-viewer/8db2adce332948cea5381ddc09105d6f). But didn't works...